app/styles/common.less

```diff
@@ -46,7 +46,20 @@ a {
   cursor: pointer;
 }
 
-// 
+li {
+  margin: 10px 0px;
+}
+
+code {
+  padding: 2px 4px;
+  font-size: 90%;
+  color: #c7254e;
```

Comment:
It's currently the same across both themes but thats a good idea - i'll include it. Can't hurt.